packages/embark-test-runner/src/index.js

```diff
@@ -235,9 +235,20 @@ class TestRunner {
                 if (global.embark.needConfig) {
                   global.config({});
                 }
-                global.embark.onReady((_err, accounts) => {
+
+                function runDescribe(accounts) {
                   self.ogMochaDescribe(describeName, callback.bind(mocha, accounts));
+                  setImmediate(() => {
+                    global.run(); // Call `run` in setImmediate to make sure tests with no `config()`
```

Comment:
I think the comment here is unfinished? What are we trying to make sure exactly? :D

Reply:
Oops, nice catch